packages/mdc-ripple/util.ts

```diff
@@ -34,7 +34,9 @@ function detectEdgePseudoVarBug(windowObj: Window): boolean {
   const document = windowObj.document;
   const node = document.createElement('div');
   node.className = 'mdc-ripple-surface--test-edge-var-bug';
-  document.body.appendChild(node);
+  // Append to head instead of body because this script might be invoked in the
+  // head, in which case the body doesn't exist yet. The probe works either way.
+  document.head.appendChild(node);
```

Comment:
Is this detection still required in latest version of Edge?

Reply:
Nope, the test returns false in Edge 18.